upgrade/upgrade-1.3.0.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Gett\MyparcelBE\Database\Table;
+
+/**
+ * @throws \PrestaShopDatabaseException
+ */
+function upgrade_module_1_3_0(): bool
+{
+    $deliverySettingsTable = Table::withPrefix(Table::TABLE_DELIVERY_SETTINGS);
+    $carrierConfiguration  = Table::withPrefix(Table::TABLE_CARRIER_CONFIGURATION);
+
+    $extraOptions = <<<SQL
+ALTER TABLE $deliverySettingsTable ADD COLUMN extra_options text;
+SQL;
+
+    $carrierConfig = <<<SQL
```

Comment:
Ik snap ergens wel dat je ze in 2 losse variabelen wil hebben, maar vernoem de variabelen dan naar wat de queries precies doen

Reply:
Dit gaat helemaal aangepast worden